src/data/en/index.ts

```diff
@@ -1179,13 +1174,16 @@ export const loggedInData = {
           label: 'Exercise type',
           subtasksTitleSummary: 'Subtasks',
           subtasksTitle: 'Should there be subtasks?',
+          subtasksTitleExerciseGroup: 'How many subtasks should there be?',
           noSubtasks: 'No',
           yesSubtasks: 'Yes',
+          subtasksLabel: 'Exercises',
           numberOfSubtasksPlaceholder: 'Number of subtasks',
           chooseOption: "Choose an option",
           multipleChoice: "Multiple Choice",
           singleChoice: "Single Choice",
-          solutionWithOneNumber: "Solution with 1 number"
+          inputExercise: "Input exercise",
```

Comment:
Those strings already exist in the editor strings so we could avoid a duplication here.

Reply:
they do… search for "Single-choice" and "Multiple-choice"